WebApp/autoreduce_webapp/instrument/views/runs.py

```diff
@@ -272,7 +272,7 @@ def configure_new_runs_post(request, instrument_name, start=0):
             # Makes the variables that will be active for the range END + 1 -> onwards
             InstrumentVariablesUtils.find_or_make_variables(possible_variables, instrument.id, post_range_args, end + 1)
 
-    elif experiment_reference != 0:
```

Comment:
I think we should leave this in, and have an else that throws a RuntimeError, because if this gets POSTed without any info about start/end or experiment reference run then something has gone wrong!

Reply:
We cant leave this in. It is referenced before assignment

Reply:
I commented above about this

Reply:
I commented this above, if you didn't see